.maintain/node-template-release/src/main.rs

```diff
@@ -233,20 +250,28 @@ fn main() {
 		// Check if this is the top level `Cargo.toml`, as this requires some special treatments.
 		if top_level_cargo_toml_path == *t {
 			// All workspace member `Cargo.toml` file paths.
-			let workspace_members = cargo_tomls.iter()
-				.filter(|p| **p != top_level_cargo_toml_path)
-				.collect();
+			let workspace_members =
+				cargo_tomls.iter().filter(|p| **p != top_level_cargo_toml_path).collect();
 
 			update_top_level_cargo_toml(&mut cargo_toml, workspace_members, &node_template_path);
 		}
 
 		write_cargo_toml(&t, cargo_toml);
 	});
 
+	// ensures rustfmt is consistent across repositories.
+    	let node_template_rustfmt_toml_path = node_template_path.join("rustfmt.toml");
+    	let rustfmt_target = Path::new("../../rustfmt.toml");
```

Comment:
This needs to be relative to `options.node_template`

Reply:
Thanks, @bkchr.  Perhaps I misunderstand the issue here.

I thought the `options.node_template` didn't have the canonical `rustfmt.toml` and therefore required the Substrate repo's `rustfmt.toml`.  In which case, there isn't a guaranteed way to get to the `substrate/rustfmt.toml` from the `options.node_template` path.

Reply:
Okay, I believe I understand my confusion.  When `node-template` is referenced in this issue, it's speaking of `bin/node-template` and therefore exists in this repo.  I mistakenly thought it was the [devhub node-tempate](https://github.com/substrate-developer-hub/substrate-node-template) from another (related) repo passed in as an argument.

Running some QA on the fix and will commit soon.